### PR TITLE
improve license detection

### PR DESCRIPTION
**Pull request by waldyrious**

### CODE REVIEW

File: src/constants.jl [hunk 1]
```diff
@@ -13,26 +13,34 @@ const LICENSES=[("MIT",[
...
+                ("GNU Affero",  [r"gnu affero general public license"]),
+                ("Apache",      [r"apache"]),
+                ("zlib",        [r"zlib", r"the origin of this software must not be misrepresented"]),
+                ("ISC",         [r"permission to use, copy, modify, and/or distribute this software"]),
+                ("Unlicense",   [r"unlicense"])
                 ]
 
 # Possible locations of licenses
-const LICFILES=["LICENSE", "LICENSE.md", "License.md", "LICENSE.txt", "LICENSE.rst",
+const LICFILES=["LICENSE", "LICENSE.md", "License.md", "LICENSE.txt", "LICENSE.rst", "UNLICENSE"
```
Comment (IainNZ) [inline src/constants.jl]: Missing comma at end of this line

File: src/constants.jl [hunk 2]
```diff
@@ -13,26 +13,34 @@ const LICENSES=[("MIT",[
...
+                "LICENCE", "LICENCE.md", "Licence.md", "LICENCE.txt", "LICENCE.rst", "UNLICENCE"
```
Comment (IainNZ) [inline src/constants.jl]: And this line

**Comment by IainNZ** [comment]
Have you done any testing? Do any Julia packages use these?

**Comment by waldyrious** [comment]
I didn't test -- I figured the changes were pretty straightforward, especially since very little actual regex is used in these patterns. And yes, all of these were from packages showing up as "Unknown" in packages.julialang.org (I checked all of the 0.3 ones).

**Comment by IainNZ** [comment]
Looks good, thanks!

**Pull request merged**